packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -259,28 +259,6 @@ describe('BlocksEditor toolbar', () => {
     await user.click(screen.getByRole('option', { name: 'Image' }));
 
     expect(screen.getByText(title)).toBeInTheDocument();
-
-    expect(baseEditor.children).toEqual([
-      {
-        type: 'image',
-        children: [
-          {
-            type: 'text',
-            text: 'A line of text in a paragraph.',
-          },
-        ],
-      },
-      // As its the last block in the editor new empty block is added below it
-      {
```

Comment:
this wont work now, as its executed after image insertion. We can improve in future PR

Reply:
I think we can't test this as we have mocked medialibrarycomponent, otherwise we will need to test the actual component, but it must be unit tested in its on scope. So I believe we can skip this test for image here. Let me know if you have alternatives to test medialibrarycomponent.